delegation/src/Pos/Delegation/Cede/Logic.hs

```diff
@@ -52,6 +53,8 @@ getPskChainInternal toIgnore issuer =
     trav x | HS.member x toIgnore = pass
     trav x = do
         whenM (uses _2 $ HS.member x) $
+            -- FIXME @volhovm We should throw something else, we don't
```

Comment:
FixME?

Reply:
It's for the future, not related to this particular PR. Probably will be fixed when we refining exceptions. In 2019 or something.